webapp/src/dogma/features/repo/RepoTokenList.tsx

```diff
@@ -1,28 +1,34 @@
-import { DeleteIcon } from '@chakra-ui/icons';
-import { Button } from '@chakra-ui/react';
+import { Text, VStack } from '@chakra-ui/react';
 import { ColumnDef, createColumnHelper } from '@tanstack/react-table';
 import { DateWithTooltip } from 'dogma/common/components/DateWithTooltip';
 import { UserRole } from 'dogma/common/components/UserRole';
-import { DynamicDataTable } from 'dogma/common/components/table/DynamicDataTable';
+import { DataTableClientPagination } from 'dogma/common/components/table/DataTableClientPagination';
 import { RepoTokenDetailDto } from 'dogma/features/repo/RepoTokenDto';
 import { useMemo } from 'react';
+import { useDeleteTokenMemberMutation } from 'dogma/features/api/apiSlice';
+import { DeleteMember } from 'dogma/features/repo/DeleteMember';
 
 export type RepoTokenListProps<Data extends object> = {
   data: Data[];
+  projectName: string;
 };
 
-const RepoTokenList = <Data extends object>({ data }: RepoTokenListProps<Data>) => {
+const RepoTokenList = <Data extends object>({ data, projectName }: RepoTokenListProps<Data>) => {
```

Comment:
How about `AppTokenList`?

Reply:
Let's handle naming separately as well. I think we may need to update the other tabs.